uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/annual-entitlements.html

```diff
@@ -0,0 +1,36 @@
+<div class="panel panel-default chr_annual-entitlements">
+  <div class="panel-heading">
+    <h3 class="panel-title">Annual Entitlements</h3>
+  </div>
+  <crm-loading show="entitlements.loaded.absencePeriods">
+    <div class="table-responsive" ng-show="entitlements.loaded.absencePeriods && entitlements.absencePeriods.length > 0">
+      <table class="table">
+        <thead>
+          <tr>
+            <th class="chr_annual-entitlements__period">Period</th>
+            <th ng-repeat="absenceType in entitlements.absenceTypes">{{absenceType.title}}</th>
+            <th class="chr_annual-entitlements__edit-entitlements"></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr ng-repeat="absencePeriod in entitlements.absencePeriods">
+            <td>{{absencePeriod.period}} <i class="fa fa-clock-o text-primary" aria-hidden="true"></i></td>
+            <td ng-repeat="absence in absencePeriod.absences">
+              {{absence.amount}}
+              <i
+                class="fa fa-exclamation-circle text-danger chr_annual-entitlements__button-icon"
+                ng-if="absence.comment"
+                ng-click="entitlements.showComment(absence.comment)"
+                aria-hidden="true">
+              </i>
+            </td>
+            <td><a href="/civicrm/admin/leaveandabsences/periods/manage_entitlements?cid={{entitlements.contactId}}&returnUrl=%2Fcivicrm%2Fcontact%2Fview%3Fcid%3D{{entitlements.contactId}}%26selectedChild%3Dabsence">Edit Entitlements</a></td>
```

Comment:
To be honest this is not a good way to specify the URL, see how it is done in jobcontract
https://github.com/civicrm/civihr/blob/c624ab2ac5c68cdc0292a32f80f2b4dee366644f/hrjobcontract/js/src/job-contract/services/utils.js#L232-L232
A similar function should be created.